<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>641</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QLabel" name="currentPageLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="fileNameLineEdit">
        <property name="text">
         <string>Enter file name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="docTypeDropdownBox">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Choose file type</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Insurance Auth</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OrthoK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Outside Rx</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>POF Waiver</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rx Request</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Referrals</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Summaries</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="prevPageButton">
        <property name="text">
         <string>Previous Page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save Page as PDF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextPageButton">
        <property name="text">
         <string>Next Page</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="clearAllButton">
        <property name="text">
         <string>Clear All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveAllPagesButton">
        <property name="text">
         <string>Save All Pages</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
